src/tests/ftest/recovery/pass_1.py

```diff
@@ -221,17 +242,16 @@ def test_orphan_pool_trus_ms(self):
         :avocado: tags=Pass1Test,test_orphan_pool_trust_ms
         """
         errors = []
-        # Run step 1 to 6 with the policies to trust MS during dmg check start.
         errors = self.run_checker_on_orphan_pool(
             policies="POOL_NONEXIST_ON_MS:CIA_TRUST_MS")
 
-        # 7. Verify that the orphan pool was removed.
+        # Verify that the orphan pool was destroyed.
```

Comment:
Not sure if it's a good idea to remove the test steps from the method description.